plugins/autocomplete/plugin.js

```diff
@@ -421,8 +433,8 @@
 			} else if ( keyCode == 38 ) {
 				this.model.selectPrevious();
 				handled = true;
-			// Enter key.
-			} else if ( keyCode == 13 ) {
+			// Completition keys.
+			} else if ( this.commitKeystroke == keyCode || ~CKEDITOR.tools.indexOf( this.commitKeystroke, keyCode ) ) {
```

Comment:
Just use `!== -1` instead of bitwise not operator, we're rarely using these in our codebase.

Reply:
Maybe it would be a better idea to make this.commitKeystroke always an array? So that you could use just `CKEDITOR.tools.indexOf()` here? That would simplify things a bit - just a suggestion, not an important thing.

Reply:
I was thinking about it earlier but with this change API could be a little inconsistent. Currently we are accepting `Number` and `Number[]` as an keystroke (by configuration) so if we change this property to an array we will be forced to use always `Number[]` with `autocomplete` instance. Not sure if it's big deal. WDYT @mlewand ?

Reply:
Not really, you could still accept `Number/Number[]` in `config.autocomplete_commitKeystroke` - but only `this.commitKeystroke` would always store array (`Number[]`). Meaning that you'd have to normalize it simply during `this.commitKeystroke` initialization.